src/Sulu/Bundle/AdminBundle/Resources/js/components/Navigation/item.scss

```diff
@@ -16,7 +16,7 @@ $activeBackgroundColor: $ebonyClayDark;
     position: relative;
 
     &:hover {
-        color: $java;
+        color: $shakespeare;
```

Comment:
That should actually be an own variable as well, similar to `$activeColor`. Should we maybe even reuse `$activeColor`?